packages/react-aria-components/src/Menu.tsx

```diff
@@ -225,14 +225,19 @@ function MenuItemInner<T>({item}: MenuItemInnerProps<T>) {
 
   let props: MenuItemProps<T> = item.props;
   let {isFocusVisible, focusProps} = useFocusRing();
+  // TODO: useMenuItem already has a useHover call, but doesn't return the hovered state. I could have the hook return
+  // it but other hooks don't do that. Adding another useHover here for now
+  let {hoverProps, isHovered} = useHover({
+    isDisabled: state.disabledKeys.has(item.key)
+  });
```

Comment:
Open question, erring on the side of caution for now and just adding a `useHover` here

Reply:
I think this is fine for now